account-kit/rn-signer/ios/turnkey/Data+EncodingExtensions.swift

```diff
@@ -0,0 +1,88 @@
+//
```

Comment:
I think there might be a pod that we could use or once turnkey unblocks us we might want to remove this? Or is there a reason to keep this and the hex extension as well?

Reply:
Context: Just trying to reduce the amount of iOS code we need to maintian.